DB/src/main/java/io/deephaven/db/v2/parquet/ParquetReaderUtil.java

```diff
@@ -268,11 +283,22 @@ public static void readParquetSchema(final String filePath, final ColumnDefiniti
 
         for (ColumnDescriptor column : schema.getColumns()) {
             currentColumn.setValue(column);
-            final LogicalTypeAnnotation logicalTypeAnnotation = column.getPrimitiveType().getLogicalTypeAnnotation();
-            final String colName = column.getPath()[0];
+            final PrimitiveType primitiveType = column.getPrimitiveType();
+            final LogicalTypeAnnotation logicalTypeAnnotation = primitiveType.getLogicalTypeAnnotation();
+            final String parquetColumnName = column.getPath()[0];
+            final String colName;
+            final String mappedName = readInstructions.getColumnNameMapping(parquetColumnName);
+            if (mappedName != null) {
+                colName = mappedName;
+            } else if (parquetColumnName.contains(" ")) {
```

Comment:
If you take my advice about mapping functions, this could be the default mapping function. We could also insist that mapping functions return non-null results, and do a `io.deephaven.db.tables.utils.DBNameValidator#validateColumnName` on the result.

Reply:
See comment where I quote Boog.

Reply:
I still think it would be nice to remember that you did this transformation and freak out if it creates a duplicate.

Reply:
Done.